security-openid-connect-web-authentication-quickstart/src/main/resources/META-INF/resources/index.html

```diff
@@ -117,10 +117,10 @@ <h2>What can I do from here?</h2>
         <p>If not already done, run the application in <em>dev mode</em> using: <code>mvn quarkus:dev</code>.
         </p>
         <ul>
-            <li>Add REST resources, Servlets, functions and other services in <code>src/main/java</code>.</li>
-            <li>Your static assets are located in <code>src/main/resources/META-INF/resources</code>.</li>
-            <li>Configure your application in <code>src/main/resources/META-INF/microprofile-config.properties</code>.
-            </li>
+          <li>Add REST resources, Servlets, functions and other services in <code>src/main/java</code>.</li>
+          <li>Your static assets are located in <code>src/main/resources/META-INF/resources</code>.</li>
+          <li>Configure your application in <code>src/main/resources/application.properties</code>.
+          </li>
```

Comment:
Agreed with the change but I think the indentation was correct and consistent with the rest of the file.

Reply:
fixed